includes/Modules/AdSense/Ad_Blocking_Recovery_Tag.php

```diff
@@ -81,21 +81,4 @@ protected function get_default() {
 			'error_protection_code' => '',
 		);
 	}
-
-	/**
-	 * Gets the callback for sanitizing the ad blocking recovery tag before saving.
-	 *
-	 * @since 1.6.0
-	 *
-	 * @return callable|null
-	 */
-	protected function get_sanitize_callback() {
-		return function( $option ) {
-			if ( ! is_array( $option ) || ! isset( $option['tag'] ) || ! isset( $option['error_protection_code'] ) || ! is_string( $option['tag'] ) || ! is_string( $option['error_protection_code'] ) ) {
```

Comment:
I am a bit confused by this change; the **Relevant technical choices** describes it as follows:

> This removes the `sanitize_callback` as it will be an encrypted string and can't realistically be verified.

However if that's the case, how are the remaining, similar checks in `get` and `set` still supposed to work?

https://github.com/google/site-kit-wp/blob/d1087f52ed6072ff77932166c4bff5cb1982d8de/includes/Modules/AdSense/Ad_Blocking_Recovery_Tag.php#L33-L41

https://github.com/google/site-kit-wp/blob/d1087f52ed6072ff77932166c4bff5cb1982d8de/includes/Modules/AdSense/Ad_Blocking_Recovery_Tag.php#L52-L58

These duplicate checks also point to the fact a bit of DRY would be appropriate here, but that's a side issue...

Reply:
As par our slack conversation, we're using an `Encrypted_Options` when instantiating this, so the `get` and `set` will work with the unencrypted values while by the time `get_sanitize_callback` is run (just before data is handed off to DB) the value has already been converted to a encrypted string. 

Good point about the DRY, I have pushed a commit to extract it out in a separate method. Cheers.

Reply:
Thanks for the clarification, and the update @kuasha420!